data/pirate jobs.txt

```diff
@@ -1372,7 +1372,7 @@ mission "Slave Transport [0]"
 	job
 	repeat
 	description "These pirates need <bunks> slaves moved to <destination> along with <cargo>. Payment is <payment>. Slavery is highly illegal throughout the galaxy, so this may be a dangerous job to take."
-	passengers 2 10 .5
+	passengers 2 10 .9
 	cargo "shackles and grub" 1
 		illegal -1
```

Comment:
Yeah, it's for outfits, as "illegal -1" was originally intended for mission cargo, so "atrocity" is what outfits use.